### PR TITLE
Korean translation

### PR DESCRIPTION
**Pull request by mirosverse**
issue #28

### CODE REVIEW

**Comment by humphd** [comment]
cc @suhhee1011, in case you want to review (no pressure).

File: README_ko-KR.md [hunk 1]
```diff
@@ -0,0 +1,1021 @@
...
+  
+
+## 머리말
+
+이것은 인공지능에 대한 *배경지식이 없는* 프로그래머들을 위한 머신러닝 **실습 가이드**입니다. 
+인공지능 신경망을 사용하는 것은 박사학위를 필요로 하지 않으며, 여러분은 크게 발전할 필요없이
+이미 있는 것을 *사용*하면 됩니다. 지금 우리가 가지고 있는 것은 충분히 유용합니다. 저는 이것을 
+다른 오픈소스를 대하는 것처럼 더 많은 사람들이 갖고 놀아야 한다고 생각합니다.   
+
+이 가이드에서 우리의 목표는 기계학습을 이용하여 [데이터/훈련되지않은 샘플들](data/untrained-samples)
```
Comment (suhhee1011) [inline README_ko-KR.md]: 기계학습보다는 머신러닝이라고 쓰는건 어떨까요? 한국에서 기계학습이라고 번역하는것보다는 머신러닝이라고 부르는경우가 더 많은 것 같아요 ㅎㅎ

File: README_ko-KR.md [hunk 2]
```diff
@@ -0,0 +1,1021 @@
...
+
+소개가 끝났으니, 여러분에게 자전거 트릭을 몇 가지 보여드리겠습니다!
+
+## 개요
+
+지금부터 살펴볼 내용은 다음과 같습니다:
+
+* 특히 기존의 오픈 소스 머신러닝 기술을 설정하고 사용합니다.([Caffe](http://caffe.berkeleyvision.org/)와
+ [DIGITS](https://developer.nvidia.com/digits))
+* 이미지 데이터셋를 만듭니다.
```
Comment (suhhee1011) [inline README_ko-KR.md]: 를->을

File: README_ko-KR.md [hunk 3]
```diff
@@ -0,0 +1,1021 @@
...
+* 신경망을 처음부터 훈련시킵니다.
+* 본 적 없는 이미지로 신경망을 테스트합니다.
+* 기존 신경망을 하게 튜닝 -*fine Tuning*- 해 신경망의 정확성을 향상시킵니다. (AlexNet 와 GoogLeNet)
```
Comment (suhhee1011) [inline README_ko-KR.md:43]: "하게" 앞에 뭔가 빠진것 같네요 ㅎㅎ

File: README_ko-KR.md [hunk 4]
```diff
@@ -0,0 +1,1021 @@
...
+를 사용할 것입니다.(BSD licensed)
+
+> Q: “잠깐만요, 왜 Caffe죠? Tensorflow와 같은 것을 사용하는 것은 어떨까요?
+> 요즘 모두가 말하는 것이잖아요...”  
+
+좋은 선택지가 많이 있고, 여러분은 모든 선택지를 살펴봐야 합니다. [TensorFlow](https://www.tensorflow.org/)는
+훌륭하고 여러분은 TensorFlow를 사용해도 좋습니다. 하지만 전 여러가지 이유로 Caffe를 사용하고 있습니다:
+
+* 컴퓨터 비전 문제에 적격입니다. 
+* C++, 파이썬을 지원합니다.([node.js 지원](https://github.com/silklabs/node-caffe) 예정)
```
Comment (suhhee1011) [inline README_ko-KR.md]: C++,python ->C++과 python

File: README_ko-KR.md [hunk 5]
```diff
@@ -0,0 +1,1021 @@
...
+미세하게 조정하는 것 -*Fine Tuning*-입니다. 아래에서 이 작업을 어떻게 하는 지 알아보겠지만, 제가 여러분에게 보여드릴 것은 최신 GPU가 탑재되지 않은 1년 
+된 맥북 프로를 사용하고 있습니다. 
+
+이와는 별도로, 전 통합 인텔 그래픽 카드와 엔비디아 GPU를 가지고 있기 때문에 [OpenCL Caffe branch]
+(https://github.com/BVLC/caffe/tree/opencl)를 사용하기로 결정했고, 제 노트북에서 잘 작동했습니다. 
+
+Caffe 설치가 완료되면 다음 작업을 수행하거나 수행해야 합니다:
+
+*  빌드된 caffe가 포함된 디렉토리입니다. 표준으로 이 작업을 수행했다면, caffe, python 바인딩 등을 실행하는
+ 데 필요한 모든 것이 `build/` 디렉터로에 있을 것입니다. `build/`의 상위 디렉토리는 `CAFFE_ROOT`(나중에 필요)입니다.
```
Comment (suhhee1011) [inline README_ko-KR.md]: 디렉터로 ->디렉터리

File: README_ko-KR.md [hunk 6]
```diff
@@ -0,0 +1,1021 @@
...
+        image_0003.jpg
+        ...
+    seahorse/
+        image_0001.jpg
+        image_0002.jpg
+        image_0003.jpg
+        ...
+```
+
+여기 각 디렉토리는 분류할 카테고리이며, 해당 카테고리 디렉토리 내의 각 이미지는 훈련 및 검증에
```
Comment (suhhee1011) [inline README_ko-KR.md]: 여기 각 디렉토리는 분류할 카테고리이며, ->여기 각 디렉토리는 이미지를 분류할 카테고리이며,

Comment (mirosverse) [reply README_ko-KR.md]: 감사합니다 눈에 안보인 오타가 생각보다 많았네요 ㅎㅎ!

**Comment by humphd** [comment]
@suhhee1011 want to take another look when you have a min?

**Review by humphd — CHANGES_REQUESTED** [review]
Can you please add a link to the main README to indicate it's available in Korean as well, like this:

https://github.com/humphd/have-fun-with-machine-learning/blob/master/README.md?plain=1#L2

Next, can you please add your name and whatever link you want for yourself on line 4, like this:

https://github.com/humphd/have-fun-with-machine-learning/blob/master/README_zh-tw.md?plain=1#L4

I can't comment on the translation itself, and I'll assume you've dealt with @suhhee1011's comments?  If not, please do that and I'll merge once you address the above.

Nice work.

File: README_ko-KR.md [hunk 7]
```diff
@@ -0,0 +1,1021 @@
...
+
+1. 분류된 이미지의 데이터셋를 구성하고 준비하십시오
+2. 신경망의 아키텍처를 규정하십시오
+3. 준비된 데이터셋를 사용해 이 신경망을 훈련시키고 검증하십시오.
+
+처음부터 시작하는 것과 사전훈련된 신경망을 사용하는 것의 차이를 보여주고 Caffe와 DIGITs에서 흔히  
+사용되는 두 가지 인기 있는 사전훈련된 신경망(AlexNet, GoogLeNet)에서 어떻게 실행하는 지 보여주기
+위해 우리는 이러한 3단계를 거칠 것입니다. 
+
+우리는 훈련 시도에 돌고래와 해마의 작은 데이터셋를 사용할 것입니다. [data/dolphins-and-seahorses](data/dolphins-and-seahorses)에 제가 사용했던 이미지들을 넣어두었습니다. 2개 이상의 카테고리가 필요하고 여러분은 더
```
Comment (suhhee1011) [inline README_ko-KR.md]: 를->을

Comment (suhhee1011) [reply README_ko-KR.md]: 혹시 이거 다른 부분도 똑같이 에러가 있던데 다 바꾸셔야 할 것 같아요 ㅎㅎ

File: README_ko-KR.md [hunk 8]
```diff
@@ -0,0 +1,1021 @@
...
+미세하게 조정하는 것 -*Fine Tuning*-입니다. 아래에서 이 작업을 어떻게 하는 지 알아보겠지만, 제가 여러분에게 보여드릴 것은 최신 GPU가 탑재되지 않은 1년 
+된 맥북 프로를 사용하고 있습니다. 
+
+이와는 별도로, 전 통합 인텔 그래픽 카드와 엔비디아 GPU를 가지고 있기 때문에 [OpenCL Caffe branch]
+(https://github.com/BVLC/caffe/tree/opencl)를 사용하기로 결정했고, 제 노트북에서 잘 작동했습니다. 
+
+Caffe 설치가 완료되면 다음 작업을 수행하거나 수행해야 합니다:
+
+*  빌드된 caffe가 포함된 디렉입니다. 표준으로 이 작업을 수행했다면, caffe, python 바인딩 등을 실행하는
+ 데 필요한 모든 것이 `build/` 디렉터로에 있을 것입니다. `build/`의 상위 디렉터리는 `CAFFE_ROOT`(나중에 필요)입니다.
```
Comment (suhhee1011) [inline README_ko-KR.md]: 디렉터로->디렉터리 아직 안바뀌어 있네요 ㅠㅠ

**Review by suhhee1011 — CHANGES_REQUESTED** [review]

**Comment by humphd** [comment]
@suhhee1011 good to merge?

**Comment by humphd** [comment]
@zaysverse looks like this got missed:

> Can you please add a link to the main README to indicate it's available in Korean as well, like this:

https://github.com/humphd/have-fun-with-machine-learning/blob/master/README.md?plain=1#L2

Can you include that too?

File: README_ko-KR.md (hunk 8 above)
Comment (mirosverse) [reply README_ko-KR.md]: 봐주셔서 정말 감사합니다. 언급하신 부분들 다시 찾아보고 수정했습니다!

**Comment by mirosverse** [comment]
I've done. I translated it while studying and it was fun. Thank you for this detailed tutorial :)

**Comment by mirosverse** [comment]
Sorry. I was a bit late because I was reviewing the whole thing finally.

**Comment by humphd** [comment]
Absent a response from @suhhee1011, I'm going to merge this.  If there are issues, people can fix in follow-ups.  Thanks a lot for all the work @zaysverse, I hope other Korean speakers benefit from your efforts!

**Comment by birkhofflee** [comment]
Hooray!

**Pull request merged**